ActiveAdminの導入

Joe-noh opened the pull request:
リソース管理にActiveAdminを使用します。

大きなカスタマイズはしておらず、パスは`/admin`、IDは`admin@example.com`、パスは`password`です。
productionに入れたあとで変えましょう。

starのフォームだけイジっていて、`status`のinputがtextだったところを、selectにしてあります。

Joe-noh committed:
bundle install

Joe-noh committed:
ActiveAdminにログインできるまで

Joe-noh committed:
perメソッドが無いエラーを解消

Joe-noh committed:
User, Relationship, Micropostを管理対象に追加

Joe-noh committed:
管理画面のlocaleは常にen

Joe-noh committed:
admin_userのfixtureはいらない

Joe-noh committed:
Starを管理対象に追加

Joe-noh committed:
管理画面のstarフォームのstatusをinput selectに変更

hanazuki commented:
![b73c92746b3a026ba649342e2f03bd5b](https://cloud.githubusercontent.com/assets/1135690/9784967/a6f31442-57e9-11e5-9a0d-a89c4afc8ca2.png)

ActiveAdminのCSS混ざってつらい現象だ

hanazuki commented:
CSS以外は問題無いと思いました！

Joe-noh committed:
CSS混入問題を解消

http://yamotonalds.hatenablog.com/entry/2014/03/18/ActiveAdmin%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%99%E3%82%8B%E5%A0%B4%E5%90%88%E3%81%AFasset%E3%81%AE%E6%B1%9A%E6%9F%93%E3%81%AB%E6%B3%A8%E6%84%8F

Joe-noh commented:
修正しました。

hanazuki commented:
:ok_woman: でも，テストおちてる

Joe-noh commented:
おお？

Joe-noh committed:
rake db:migrate:reset

Joe-noh committed:
実行時の日付で挙動が変わるテストを修正
